products/bash/release-ace-integration-server.sh

```diff
@@ -56,6 +63,10 @@ if [[ -z "$imageTag" ]]; then
   exit 1
 fi
 
+# tracing false if tracing namespace is empty
+[ -z $tracing_namespace ] && tracing_enabled="false"
```

Comment:
is tracing_enabled ever actually false here? I can't see how you'd run this script without tracing on.